.palantir/revapi.yml

```diff
@@ -1,8 +1,98 @@
+      new: null
+      justification: "this is all internal code"
+    - code: "java.method.removed"
+      old: "method com.palantir.atlasdb.factory.ImmutableLocalPaxosServices.Builder\
+        \ com.palantir.atlasdb.factory.ImmutableLocalPaxosServices.Builder::isCurrentSuspectedLeader(java.util.function.Supplier<java.lang.Boolean>)"
+      new: null
+      justification: "this is all internal code"
+    - code: "java.method.removed"
+      old: "method com.palantir.atlasdb.factory.ImmutableLocalPaxosServices com.palantir.atlasdb.factory.ImmutableLocalPaxosServices::withPingableLeader(com.palantir.leader.PingableLeader)"
+      new: null
+      justification: "this is all internal code"
+    - code: "java.method.removed"
+      old: "method com.palantir.leader.PingableLeader com.palantir.atlasdb.factory.Leaders.LocalPaxosServices::pingableLeader()"
+      new: null
+      justification: "this is all internal code"
+    - code: "java.method.addedToInterface"
+      old: null
+      new: "method java.util.Set<com.palantir.leader.PingableLeader> com.palantir.atlasdb.factory.Leaders.LocalPaxosServices::remotePingableLeaders()"
+      justification: "this is all internal code"
+    - code: "java.method.returnTypeChanged"
+      old: "method java.util.Map<com.palantir.leader.PingableLeader, com.google.common.net.HostAndPort>\
+        \ com.palantir.atlasdb.factory.Leaders::generatePingables(com.palantir.atlasdb.util.MetricsManager,\
+        \ java.util.Collection<java.lang.String>, java.util.Optional<com.palantir.conjure.java.config.ssl.TrustContext>,\
+        \ com.palantir.conjure.java.api.config.service.UserAgent)"
+      new: "method java.util.List<com.palantir.leader.PingableLeader> com.palantir.atlasdb.factory.Leaders::generatePingables(com.palantir.atlasdb.util.MetricsManager,\
+        \ java.util.Collection<java.lang.String>, java.util.Optional<com.palantir.conjure.java.config.ssl.TrustContext>,\
+        \ com.palantir.conjure.java.api.config.service.UserAgent)"
+      justification: "this is all internal code"
+    - code: "java.method.removed"
+      old: "method com.palantir.atlasdb.factory.ImmutableLocalPaxosServices.Builder\
+        \ com.palantir.atlasdb.factory.ImmutableLocalPaxosServices.Builder::pingableLeader(com.palantir.leader.PingableLeader)"
+      new: null
+      justification: "this is all internal code"
+    - code: "java.method.removed"
+      old: "method com.palantir.atlasdb.factory.ImmutableLocalPaxosServices com.palantir.atlasdb.factory.ImmutableLocalPaxosServices::withIsCurrentSuspectedLeader(java.util.function.Supplier<java.lang.Boolean>)"
+      new: null
+      justification: "this is all internal code"
+    - code: "java.method.addedToInterface"
+      old: null
+      new: "method com.palantir.leader.PingableLeader com.palantir.atlasdb.factory.Leaders.LocalPaxosServices::localPingableLeader()"
+      justification: "this is all internal code"
```

Comment:
@CRogers kinda interesting feedback here - felix ended up running `acceptAllBreaks` twice because develop was released while this PR was still in flight. What do you think about getting rid of the version number and just hashing the content of the break somehow?